src/scrabble.js

```diff
@@ -1,6 +1,305 @@
+
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.doubleLetterArray = []
+    this.tripleLetterArray = []
+    this.doubleLetterArrayReduced = []
+    this.tripleLetterArrayReduced = []
+
+    this.bracketsToRemove = ['[', ']', '{', '}']
+    this.doubleScoreLettersToRemove = []
+    this.arrayMinusDoubleScoredLetters = []
+
+    this.totalScore = 0
+
+    // this.cleanArrayForScoring
+
+    this.word = word
+    this.curly = '{'
+    this.square = '['
+    this.wordScore = 0
+    this.scoresObject = {
+      A: 1,
+      E: 1,
+      I: 1,
+      O: 1,
+      U: 1,
+      L: 1,
+      N: 1,
+      R: 1,
+      S: 1,
+      T: 1,
+      D: 2,
+      G: 2,
+      B: 3,
+      C: 3,
+      M: 3,
+      P: 3,
+      F: 4,
+      H: 4,
+      V: 4,
+      W: 4,
+      Y: 4,
+      K: 5,
+      J: 8,
+      X: 8,
+      Q: 10,
+      Z: 10
+    }
+  }
+
+  wordArray () {
+    return [...this.word]
+  }
+
+  score () {
+    if (this.word === '') {
+      return 0
+    } else if (this.word === '\t') {
+      return 0
+    } else if (this.word === null) {
+      return 0
+    } else if (this.word === '\n') {
+      return 0
+    } else if (this.word.includes(' ') || this.word.includes('\t') || this.word.includes('\n')) {
+      return 0
```

Comment:
All of these conditions are returning 0, so maybe you could condense this into one conditon.
And, what if this.word is other data types (e.g NaN, false etc)? It might pass the conditionals...
I would suggest doing 
`if(typeof this.word !== "string")`

Then you only need two lines:)